open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/auditlog/OMAGAdminAuditCode.java

```diff
@@ -86,89 +86,174 @@
                                     "resources that they were using.  Restart the server whenever its services are needed again."),
```

Comment:
Was it necessary to reformat the whole file - makes it difficult to see what has changed